src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -464,6 +466,86 @@ describe('non-empty successful affected clusters table', () => {
       .should('have.text', 'dd2ef343-9131-46f5-8962-290fdfdf2199');
   });
 
+  describe('modal for bulk disabling', () => {
+    beforeEach(() => {
+      cy.intercept(
+        'PUT',
+        '/api/insights-results-aggregator/v1/clusters/**/rules/**/error_key/**/disable',
+        {
+          statusCode: 200,
+        }
+      ).as('disableRequest');
+      cy.intercept(
+        'POST',
+        '/api/insights-results-aggregator/v1/clusters/**/rules/**/error_key/**/disable_feedback',
+        {
+          statusCode: 200,
+        }
+      ).as('disableFeedbackRequest');
+    });
+
+    it('modal for bulk disabling', () => {
+      cy.get(BULK_SELECT).find('input').click().should('be.checked');
+
+      cy.get(TOOLBAR)
+        .find('.pf-m-spacer-sm')
+        .find(DROPDOWN)
+        .within((el) => {
+          cy.wrap(el).click();
+          cy.get('button')
+            .contains('Disable recommendation for selected clusters')
+            .click();
+        });
+
+      cy.get(MODAL).find(CHECKBOX).should('be.checked');
+
+      cy.get(MODAL).find('button[data-ouia-component-id="confirm"]').click();
+      // Should catch at least one PUT and at least one POST requests after clusters rule disable
```

Comment:
I managed to count the requests as 

```cy.get('@disableRequest.all').its('length').should('equal', data['enabled'].length);```

but to make it work we need to have a condition to wait for before doing the count, so maybe it is not that feasible

Reply:
I think waiting 3-5 secs could be sufficient to count all needed requests...

Reply:
Ideally, we should have a command that tries to get X number of requests and times out if fails to do so. But probably it is not easy to do with cypress. Do you prefer to implement the 3-5 secs waiting, or we leave as is as this function is going to be modified in the near future with API v2 call?

Reply:
I would vote for ignoring it TBH, this implementation won't stay for long, and, to test this, it requires creating custom solutions... which is mainly a consequence of the current lack of our API endpoints rather than of the weak cypress inventory

Reply:
ok, fair point